### ISSUE TITLE
[youtube, windows] add additional errorlevel

### ISSUE THREAD

**Issue by KamelittaOida** [yt-dlp/yt-dlp#2947]
### Checklist

- [X] I'm reporting a feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

is it possible to add an additional errorlevel %errorlevel%?
Both states give off the same one.
```
[youtube:tab] 'chanID': Downloading webpage
ERROR: [youtube:tab] 'chanID': The channel is not currently live
>echo 1
```
```
ERROR: [youtube] vidID: This live event will begin in 15 minutes.
>echo 1
```

**Comment by pukkandan** [comment]
This is not easily possible. We would need #457 before this can even be attempted

**Comment by KamelittaOida** [comment]
:( alright
thanks

**Comment by pukkandan** [comment]
Actually, on a second look, your specific use-case is not possible. Both the errors in your issue are of the same "type". ie, both are youtube saying a specific link is unavailable. So even with #457, these will belong to the same group

**Comment by KamelittaOida** [comment]
just so we're clear. in both instances I'm inquiring https://www.youtube.com/channel/chanID/live

**Comment by pukkandan** [comment]
Yes, and in both instances, youtube is saying that the video is unavailable

**Comment by KamelittaOida** [comment]
As nooblet what I don't understand is that you have two exit codes (or whatever) from the same link, and it's not possible to create an if command?
if 'currently' then errorlevel 2

**Comment by pukkandan** [comment]
you can write a script to do so. As far as yt-dlp is concerned, the two errors are of the same type as I explained above
